content/projekte/_index.md

```diff
@@ -9,16 +9,88 @@ outputs:
 
 {{< headline-icon icon="icon-documents.svg" color="blue" >}}
 
-{{< headline-brackets-h1 color="red"  >}}
-Unsere Projekte
-{{< /headline-brackets-h1  >}}
+{{< headline-brackets-h1 color="red" >}}
+Projekte
+{{< /headline-brackets-h1 >}}
 
+{{< hero-block-center >}}
+[Alle Projekte](projekte/alle), die bei Code for Germany entstehen, sind Open Source. Sie lassen sich kostenlos verwenden und weiterentwicklen.
+{{< /hero-block-center >}}
 
-{{< hero-block-center  >}}
-In unseren Labs entstehen Projekte, die aufzeigen, welchen Mehrwert Offene Daten und Freie und Offene Software für unsere Gesellschaft bieten und wie diese Open Government begünstigen. 
-{{< /hero-block-center  >}}
+{{< paragraph-center >}}
+Seit der Gründung von Code for Germany im Jahr 2014 zeigen wir, was mit offenen Technologien alles möglich ist. Wir bauen Prototypen um selbst zu lernen; um zu zeigen, wie sich Technik zum Wohle aller einsetzen lässt; und um andere davon zu überzeugen, es uns gleich zu tun. Wir haben kein wirtschaftliches Interesse. Wir wünschen uns, dass unsere Projekte –  all unsere Apps, Plattformen, Sensoren, Prototypen, Ideen –  möglichst viel weiterverwendet werden und möglichst vielen Menschen nutzen.
```

Comment:
finde das mit den Semikolons etwas komisch. Sollten das nicht besser Kommas sein, oder eine Aufzählungsliste?

Wir bauen Prototypen 
 * um selbst zu lernen
 * um zu zeigen, wie sich Technik zum Wohle aller einsetzen lässt und 
 * um andere davon zu überzeugen, es uns gleich zu tun.

Reply:
Ich will hier gerne eine Lanze brechen für die Semikolons! Sie sind da, damit man eine kleine Pause machen kann gedanklich beim Lesen: Grammatikalisch ist das korrekt so, man kann eine Aufzählungsliste mit Semikolons machen, wenn es dadurch übersichtlicher wird als mit Kommas. [Hier nerdiger Link zur Rechtfertigung.](https://www.leemeta-uebersetzungen.de/blog/grammatik/welche-funktion-hat-das-semikolon) Bulletpoints finde ich in diesem Absatz nicht schön. Könntest du damit leben?

Reply:
Ich kann damit leben, fand es nur ungewohnt. So, rescue the Strichpunkt 😀